locales/id/validation-inline.php

```diff
@@ -90,15 +90,15 @@
         'numeric' => 'Nilai harus berisi minimal :min.',
         'string'  => 'String harus berisi minimal :min karakter.',
     ],
-    'multiple_of'          => 'The value must be a multiple of :value',
+    'multiple_of'          => 'Isian :attribute harus merupakan kelipatan dari :value.',
```

Comment:
keep in mind that the validation-inline.php file does not come with Laravel and the idea of this file is not to put a specific name to each attribute (as in validation.php) but a generic name for the validation attributes. Therefore in the translations of this file the placeholder :attribute should not appear. [View here](https://laravel-lang.github.io/lang/contributing-to-dev.html)

Reply:
![image](https://user-images.githubusercontent.com/10347617/114825844-64ced600-9dcf-11eb-895f-50387fea3480.png)